python/py-nbxmpp/Portfile

```diff
@@ -4,14 +4,14 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-nbxmpp
-version             0.6.10
+version             2.0.6
 revision            0
 categories-append   net
 platforms           darwin
```

Comment:
this is the default value for the `platforms` field since MacPorts v2.7 - please remove